website/content/en/status/report-2023-01-2023-03/pot.adoc

```diff
@@ -0,0 +1,20 @@
+=== Containers and FreeBSD: Pot, Potluck and Potman
+
+Links: +
+link:https://github.com/bsdpot[Pot organization on GitHub] URL: link:https://github.com/bsdpot[https://github.com/bsdpot]
```

Comment:
To note:

`link:https://github.com/bsdpot[https://github.com/bsdpot]`
This is redundant, don't need to repeat.
`link:https://github.com/bsdpot[]`


Could you please fix this in the template when you have time? It's coming from there.

I won't bother you with this in the following reports.

and you can proceed here.

Thanks.

Reply:
Sure, I will fix the template.

I will also remove the redundancy from all reports (at least of this quarter), but I will probably do it in one single commit for all of them at once rather than fixing them one by one as they come.